R/class_partial_time_coercion.R

```diff
@@ -1,13 +1,16 @@
 #' Coerce an object to a parttime object
 #'
 #' @param x an object for coersion
-#' @param format a \code{function} or \code{character} value. If a
-#'   \code{function}, it should accept a character vector and return a matrix of
-#'   parttime components. If a \code{character} it should provide a regular
+#' @param format a `function` or `character` value. If a
+#'   `function`, it should accept a character vector and return a matrix of
+#'   parttime components. If a `character` it should provide a regular
 #'   exprssion which contains capture groups for each of the parttime
-#'   components.  See \code{?parse_to_parttime_matrix}'s \code{regex} parameter
+#'   components.  See [parse_to_parttime_matrix]'s `regex` parameter
 #'   for more details.
-#' @param ... Additional arguments passed to \code{format} when a function is
+#' @param on.na a `function` used to signal a condition for new `NA` values
+#'   introduced by coercion, a `character` value among `"error"`, `"warning"` or
+#'   `"suppress"` (for silencing messages) or `NULL` equivalent to `"suppress"`.
```

Comment:
I did some code cleanup, so just for the sake of drawing attention to the important bits, this is the new `on.na` parameter.